docs/DeveloperGuide.md

```diff
@@ -271,6 +272,27 @@ The following activity diagram summarizes what happens when a user executes a ne
 
 ---
 
+## **Planned Enhancements**
```

Comment:
I added a Future Features in UG. Should we consolidate everything here and remove the UG part?
![image](https://github.com/user-attachments/assets/47bcc107-1bfb-4d4d-b907-9c1cf536eae2)

Reply:
I agree that consolidating everything to the DG is better.